english/slap-it-hard/teach-it-a-lesson.md

```diff
@@ -0,0 +1,16 @@
+You look at the crass marshmallow man (woman?) and give it a cold, hard stare down.
+	
+	"You, yes you! How dare you think you can take my lighter, who do you think you are, hm?"
+The marshmallow being has a queer gaze in its eyes. But it's not looking at you.
+It's looking at the lighter.
+
+	"Ah, I see. Afraid I'll light up the whole place, are you? 
+	No, no, you mustn’t think that!" You say this with a bit of sarcasm, 
+	after all, this marshmallow thingy can't possibly fathom sarcasm.
+As if it's reading your thoughts, the Marsh-thingy springs forward, it's
+delicious little legs scrunching up to get more height. It's coming for you!
+
+What do you do?
+
+[Prepare for it and stand your ground!](stand/stand.md)
+[Move aside and let it bounce to the ground.](move/move.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.